### PR TITLE
Updated BlobSamples.php to include listing all blobs with prefix

### PR DESCRIPTION
**Pull request by XiaoningLiu**

### CODE REVIEW

File: samples/BlobSamples.php [hunk 1]
```diff
@@ -326,12 +327,22 @@ function blobMetadata($blobClient)
...
+
     $content = fopen("myfile.txt", "r");
     $blob_name = "myblob";
+
+    $content2 = "string content";
+    $blob_name2 = "myblob2";
     
     try {
         //Upload blob
         $blobClient->createBlockBlob("mycontainer", $blob_name, $content);
```
Comment (yaxia) [inline samples/BlobSamples.php]: I'd suggest to put it in a variable and use it throughout the file.

File: samples/BlobSamples.php [hunk 2]
```diff
@@ -567,38 +585,25 @@ function snapshotOperations($blobClient)
...
+    if (file_exists('myblob.txt')) {
+        unlink('myblob.txt');
+    }
+    if (file_exists('PageContent.txt')) {
+        unlink('PageContent.txt');
+    }
+    if (file_exists('HelloWorldSnapshotCopy.png')) {
+        unlink('HelloWorldSnapshotCopy.png');
+    }
+    echo "Successfully cleaned up\n";
```
Comment (yaxia) [inline samples/BlobSamples.php]: Delete the container you created.

File: samples/BlobSamples.php [hunk 3]
```diff
@@ -413,12 +424,19 @@ function listBlobsSample($blobClient)
...
         // List blobs.
-        $blob_list = $blobClient->listBlobs("mycontainer");
-        $blobs = $blob_list->getBlobs();
-    
-        foreach ($blobs as $blob) {
-            echo $blob->getName().": ".$blob->getUrl().PHP_EOL;
-        }
+        $listBlobsOptions = new ListBlobsOptions();
+        $listBlobsOptions->setPrefix("myblob");
+        $listBlobsOptions->setMaxResults(1);
```
Comment (yaxia) [inline samples/BlobSamples.php]: Better to have a comment to clarify that setting max result to 1 is just to demonstrate the continuation token. It is not the recommended value in product.

File: samples/BlobSamples.php (hunk 1 above)
Comment (XiaoningLiu) [reply samples/BlobSamples.php]: Sovled

File: samples/BlobSamples.php (hunk 2 above)
Comment (XiaoningLiu) [reply samples/BlobSamples.php]: Sovled

File: samples/BlobSamples.php (hunk 3 above)
Comment (XiaoningLiu) [reply samples/BlobSamples.php]: done

**Pull request merged**